jre/product.yml

```diff
@@ -5,3 +5,24 @@ type: product
 benchmark_root: "./guide"
 
 profiles_root: "./profiles"
+
+cpes:
+  jre-oracle:
+    name: "cpe:/a:oracle:jre:"
+    title: "Oracle's Java Runtime Environment"
+    check_id: installed_app_is_java
```

Comment:
do we need `jre-oracle`? Why not skip it and start with `- name`?

Reply:
The product CPES don't need to have these `jre-oracle` and `jre-ibm`... They are like IDs for the CPEs, but are not actually used by the build system.

But, the IDs of the CPEs in `shared/applicability/cpes.yml` are needed, these are IDs are referenced by rules and profiles.
I just tried to keep same declaration format.

Reply:
> The product CPES don't need to have these `jre-oracle` and `jre-ibm`... They are like IDs for the CPEs, but are not actually used by the build system.
> 
> But, the IDs of the CPEs in `shared/applicability/cpes.yml` are needed, these are IDs are referenced by rules and profiles.
> I just tried to keep same declaration format.

So, if I am understanding you correctly, `jre-oracle`, `jre-ibm`, etc. aren't needed by the build system in `product.yml` but are needed by the build system in `cpes.yml`?

Reply:
Well, the IDs are there for any content to refer to them.
But as they are already declared in the `product.yml` they are for, nothing else references them.
Rules and profiles refer to the IDs of `cpes` declared in `cpes.yml`.

I'd rather keep the syntax the same for `cpes` in `product.yml` and `cpes.yml`.

Reply:
Actually, the CPE IDs are needed.
The `ocp4` profiles reference CPE's defined in the `product.yml`.